src/SFA.DAS.EmployerIncentives.Api/Types/ValidationType.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace SFA.DAS.EmployerIncentives.Api.Types
+{
+    public enum ValidationType
+    {
```

Comment:
Think this enum should just include:
HasDaysInLearning
HasNoDataLocks
IsInLearning
EmployedBeforeSchemeStarted
EmployedAtStartOfApprenticeship

Anything else should throw an error

Reply:
Updated